### DISCUSSION TITLE
Update d3-selection in react-charting

### DISCUSSION THREAD

**Discussion by dingyi1995** [microsoft/fluentui#24847] [Q&A] [answered]
Hi team, my project uses react-charting package to dome some data visualization. Recently, I try to use react flow to draw some dependency graph in my project. However, I hit an error: **TypeError: selection.interrupt is not a function**. The root cause is that I installed two versions of d3-selection in the project. 
react-charting depends on d3-selection v1.3.0, react-flow depends on d3-selection v3.0.0.
Do you team have plan to update the d3-selection package to make it compatible with other packages that uses higher version of d3-selection?

**Comment by AtishayMsft** [comment, marked as answer]
@dingyi1995 We do have plans to upgrade the d3 dependencies for the react-charting package in coming months.
